onnxruntime/core/session/provider_registration.cc

```diff
@@ -68,7 +68,7 @@ ORT_API_STATUS_IMPL(OrtApis::SessionOptionsAppendExecutionProvider,
 
   if (strcmp(provider_name, "QNN") == 0) {
 #if defined(USE_QNN)
-    options->provider_factories.push_back(QNNProviderFactoryCreator::Create(provider_options));
+    options->provider_factories.push_back(QNNProviderFactoryCreator::Create(provider_options, &(options->value)));
```

Comment:
>QNNProviderFactoryCreator [](start = 42, length = 25)

Need to updateonnxruntime_pybind_state.cc

Reply:
Thanks. Updated